static/src/js/components/TemporalDisplay/TemporalSelectionDropdown.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-underscore-dangle */
```

Comment:
I get moving this up reduces copying but, I think that when we ignore a `linting` rule its good to just have that next to where we are breaking the rule

Reply:
I agree, ignoring a linting rule for the whole file is an easy way to ignore the rule in the future without realizing it. Every time we want to ignore a rule we need to make a conscious decision to do it, so only use `eslint-disable-next-line`